c/polyjuice_errors.h

```diff
@@ -17,5 +17,6 @@
 #define ERROR_BALANCE_OF_ANY_SUDT          -87
 #define ERROR_TRANSFER_TO_ANY_SUDT         -88
 #define ERROR_RECOVER_ACCOUNT              -89
+#define ERROR_ETH_TO_GODWOKEN_ADDR         -90
```

Comment:
https://github.com/nervosnetwork/godwoken-polyjuice/blob/647d9bcd3daf376f1bfeab93f4ec461f123eb7e0/c/polyjuice_utils.h#L184

Are we need to adjuest this function?